src/storage/v2/disk/rocksdb_storage.hpp

```diff
@@ -0,0 +1,84 @@
+// Copyright 2023 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#pragma once
+
+#include <rocksdb/comparator.h>
+#include <rocksdb/db.h>
+#include <rocksdb/iterator.h>
+#include <rocksdb/options.h>
+#include <rocksdb/status.h>
+#include <rocksdb/utilities/transaction_db.h>
+
+#include "storage/v2/id_types.hpp"
+#include "storage/v2/property_store.hpp"
+#include "utils/logging.hpp"
+
+namespace memgraph::storage {
+
+/// TODO: this should be somehow more wrapped inside the storage class so from the software engineering perspective
+/// it isn't great to have this here. But for now it is ok.
+/// Wraps RocksDB objects inside a struct. Vertex_chandle and edge_chandle are column family handles that may be
+/// nullptr. In that case client should take care about them.
+struct RocksDBStorage {
```

Comment:
Define copy ctor, copy assignment, move ctor and move assignment

Reply:
as move ctor and move assignment are deleted, and others are defaulted, and I don't know what do we want to permit